clis/ph-cli/src/services/vetra.ts

```diff
@@ -118,69 +111,22 @@ async function startLocalVetraSwitchboard(
   }
 }
 
-async function spawnConnect(
-  options?: ConnectStudioOptions & { verbose?: boolean; connectPort?: number },
-  localReactorUrl?: string,
-) {
-  const { verbose = false, connectPort, ...connectOptions } = options || {};
-
-  const finalOptions = connectPort
-    ? { ...connectOptions, port: connectPort.toString() }
-    : connectOptions;
-
-  const child = fork(
-    path.join(dirname(__dirname), "commands", "connect.js"),
-    ["spawn", JSON.stringify(finalOptions)],
-    {
-      silent: true,
-      env: {
-        ...process.env,
-        PH_CONNECT_DEFAULT_DRIVES_URL: localReactorUrl,
-        PH_CONNECT_DRIVES_PRESERVE_STRATEGY: "preserve-all",
-      },
-    },
-  ) as ChildProcessWithoutNullStreams;
-
-  return new Promise<void>((resolve) => {
-    let connectInitialized = false;
-
-    child.stdout.on("data", (data: Buffer) => {
-      const message = data.toString();
-      const lines = message.split("\n").filter((line) => line.trim().length);
+type VetraConnectOptions = {
+  port?: number;
+  disableLocalPackages?: boolean;
+  vetraDriveUrl: string;
+};
 
-      if (!connectInitialized) {
-        resolve();
-        connectInitialized = true;
-        if (!verbose) {
-          console.log(green(`[Connect]: Connect initialized`));
-        }
-      }
+const startLocalConnect = (options: VetraConnectOptions) => {
+  process.env.PH_CONNECT_DEFAULT_DRIVES_URL = options.vetraDriveUrl;
+  process.env.PH_CONNECT_DRIVES_PRESERVE_STRATEGY = "preserve-all";
 
-      for (const line of lines) {
-        if (verbose) {
-          process.stdout.write(green(`[Connect]: ${line}\n`));
-        } else {
-          // Only show Local and Network URLs when not verbose
-          if (line.includes("➜  Local:") || line.includes("➜  Network:")) {
-            process.stdout.write(green(`[Connect]: ${line}\n`));
-          }
-        }
-      }
-    });
-
-    child.stderr.on("data", (data: Buffer) => {
-      if (verbose) {
-        process.stderr.write(red(`[Connect]: ${data.toString()}`));
-      }
-    });
+  if (options.disableLocalPackages) {
+    process.env.DISABLE_LOCAL_PACKAGES = "true";
```

Comment:
used this to disable load the local packages in connect

Reply:
Let's use the `PH_` prefix on our env variables. If they also need to be available on the browser for Connect then they need to be `PH_CONNECT`. In this case it's only needed at runtime.

Instead of setting the env variables for Connect here, I would pass it as options to `startConnect`, where we can centralize the logic to go from options to env vars.

Reply:
fixed in [08cb25d](https://github.com/powerhouse-inc/powerhouse/pull/1907/commits/08cb25df21956371cb5e947daf2abffe067cc47d)